run_ci.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -ex
 
+# grabs files and puts them into $LOGDIR to be saved as jenkins artifacts
+function getlogs(){
+    LOGDIR=/home/notstack/dev-scripts/logs
+
+    # Grab the host journal
+    sudo journalctl > $LOGDIR/bootstrap-host-system.journal
+}
```

Comment:
```suggestion
    export KUBECONFIG=ocp/auth/kubeconfig
    oc --request-timeout=5s get clusterversion/version || true
    oc --request-timeout=5s get clusteroperators || true
    oc --request-timeout=5s get pods --all-namespaces | grep -v Running | grep -v Completed || true
}
```

Reply:
Will add, I guess I need to include "--config ocp/auth/kubeconfig " in each of those?

Reply:
Its not necessary w/ KUBECONFIG env var set

Reply:
ahh, thats why I needed it